src/index.css

```diff
@@ -153,3 +153,10 @@ a[disabled] {
 .feedback-form .text-underline:hover {
   text-decoration: none;
 }
+
+@media only screen and (max-device-width: 312px) {
```

Comment:
This doesn't catch every size. The line still wraps at screen widths 313px - 320px.
<img width="326" alt="screen shot 2018-08-23 at 8 57 31 am" src="https://user-images.githubusercontent.com/2153230/44526819-2252ab00-a6b3-11e8-8881-614eff7e6bfb.png">

Reply:
Since the buttons should be center aligned all the time anyways. You can get rid of the media query rule (Line #157) and keep the declaration.

Reply:
Sounds good, updated